Monika After Story/game/screens.rpy

```diff
@@ -3,34 +3,30 @@ init 100 python:
     layout.UNSTABLE = store.mas_layout.UNSTABLE
 
 init -1 python:
-    layout.QUIT_YES = "Please don't close the game on me!"
-    layout.QUIT_NO = "Thank you, [player]!\nLet's spend more time together~"
+    layout.QUIT_YES = _("Please don't close the game on me!")
+    layout.QUIT_NO = _("Thank you, [player]!\nLet's spend more time together~")
 
     # tooltips
-    layout.MAS_TT_SENS_MODE = (
-        "Sensitive mode removes content that may be disturbing, offensive, "
-        " or considered tasteless."
+    layout.MAS_TT_SENS_MODE = _(
+        "Sensitive mode removes content that may be disturbing, offensive, or considered tasteless."
```

Comment:
long string lines are split to keep the character line limit to 80-120 characters. please split these lines like they were before.